cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -450,6 +453,58 @@ instance ( Condense (HeaderHash blk)
   -- user defined formatting of log output
   trTransformer _ verb tr = trStructured verb tr
 
+instance DefinePrivacyAnnotation a => DefinePrivacyAnnotation (TraceLabelPeer peer a)
+instance DefineSeverity a => DefineSeverity (TraceLabelPeer peer a)
+
+instance (Transformable Text IO a) => Transformable Text IO (TraceLabelPeer peer a) where
+  trTransformer f v t = Tracer $ \(TraceLabelPeer _peer a) ->
+    traceWith (trTransformer f v t) a
+instance DefinePrivacyAnnotation (TraceSendRecv (BlockFetch blk))
+instance DefineSeverity (TraceSendRecv (BlockFetch blk)) where
+  defineSeverity _ = Debug
+
+instance ( Show blk
+         , StandardHash blk
+         , ToObject (AnyMessage (BlockFetch blk))
+         )
+ => Transformable Text IO (TraceSendRecv (BlockFetch blk)) where
+  trTransformer = defaultTextTransformer
+
+instance ( ToObject (AnyMessage ps)
+         )
+ => ToObject (TraceSendRecv ps) where
+  toObject verb (TraceSendMsg m) = mkObject
+    [ "kind" .= String "Send" , "msg" .= toObject verb m ]
+  toObject verb (TraceRecvMsg m) = mkObject
+    [ "kind" .= String "Recv" , "msg" .= toObject verb m ]
+
+instance ( Condense (HeaderHash blk)
+         , Condense (TxId (GenTx blk))
+         , HasHeader blk
+         , HasTxs blk
+         , HasTxId (GenTx blk)
+         )
+ => ToObject (AnyMessage (BlockFetch blk)) where
+  toObject _verb (AnyMessage (MsgBlock blk)) =
+    mkObject
+      [ "kind" .= String "MsgBlock"
+      , "blkid" .= String (pack . condense $ blockHash blk)
+      , "txids" .= Array (V.fromList . fmap presentTx $ extractTxs blk)
```

Comment:
the newly introduced dependency "vector" is used only once, exactly here. Is this necessary?
a list of "String"s would also be acceptable. we do not need "Array" here.

Reply:
@CodiePP, I'm sorry, I don't understand -- there isn't an Aeson type with a `[]` -- only one with a vector.

Reply:
There isn't an Aeson Value ADT variant with a [] -- only one with a vector -- but we can reuse `toJSON` to avoid the dependency

Good point.